tsconfig.json

```diff
@@ -1,13 +1,17 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2018",
     "module": "commonjs",
     "strict": true,
     "outDir": "dist",
     "sourceMap": true,
+    "esModuleInterop": true,
```

Comment:
If I don't add this, the generated input type have silly names: "class_1" instead of "ExampleFilter" and "class_2" instead of "ExampleCondition"